### ISSUE TITLE
Access to the path is denied on terminal server when more users run the Bravo

### ISSUE THREAD

**Issue by ZdenekSoldan** [sql-bi/Bravo#590]
### Describe the problem

On the terminal server (Citrix) when more then one user run the app the second user receives the error: 

```
System.UnauthorizedAccessException: Access to the path is denied.
   at System.IO.Pipes.NamedPipeServerStream.Create(String pipeName, PipeDirection direction, Int32 maxNumberOfServerInstances, PipeTransmissionMode transmissionMode, PipeOptions options, Int32 inBufferSize, Int32 outBufferSize, PipeSecurity pipeSecurity, HandleInheritability inheritability, PipeAccessRights additionalAccessRights)
   at System.IO.Pipes.NamedPipeServerStreamAcl.Create(String pipeName, PipeDirection direction, Int32 maxNumberOfServerInstances, PipeTransmissionMode transmissionMode, PipeOptions options, Int32 inBufferSize, Int32 outBufferSize, PipeSecurity pipeSecurity, HandleInheritability inheritability, PipeAccessRights additionalAccessRights)
   at Sqlbi.Bravo.Infrastructure.AppInstance.StartPipeServer(PipeSecurity pipeSecurity) in D:\a\1\s\src\Infrastructure\AppInstance.cs:line 118
   at Sqlbi.Bravo.Infrastructure.AppInstance..ctor() in D:\a\1\s\src\Infrastructure\AppInstance.cs:line 38
   at Sqlbi.Bravo.Program.Main() in D:\a\1\s\src\Program.cs:line 20
```

### Bravo version

1.0.1.

### Tabular model

Power BI Desktop, import mode

### Power BI

 2.110.1085.0 64-bit (October 2022)

**Comment by albertospelta** [comment]
Hi @ZdenekSoldan, we have an internal build available that tries to fix the error.
It would be great if you could test it and let us know if the issue still persists. 

The build ID is `20230203.1-1.0.8434.23674` and you can download it from here https://bravo.bi/builds or by changing the update channel from `Stable` to `Dev` in the Bravo options panel and then follow the download link.

**Comment by ZdenekSoldan** [comment]
Hi @albertospelta 

> Hi @ZdenekSoldan, we have an internal build available that tries to fix the error. It would be great if you could test it and let us know if the issue still persists.
> 
> The build ID is `20230203.1-1.0.8434.23674` and you can download it from here https://bravo.bi/builds or by changing the update channel from `Stable` to `Dev` in the Bravo options panel and then follow the download link.

Hi @albertospelta this build is working as expected!

**Comment by albertospelta** [comment]
Fixed in version [1.0.2](https://github.com/sql-bi/Bravo/releases/tag/v1.0.2)
